src/main/java/aquality/appium/mobile/actions/IActionsModule.java

```diff
@@ -0,0 +1,14 @@
+package aquality.appium.mobile.actions;
+
+/**
+ * Describes implementations of actions services to be registered in DI container.
+ */
+public interface IActionsModule extends aquality.selenium.core.configurations.IConfigurationsModule {
```

Comment:
Can we user import instead of full name `aquality.selenium.core.configurations.IConfigurationsModule` ?